SimCalorimetry/HGCalSimAlgos/src/#HGCalSiNoiseMap.cc#

```diff
@@ -0,0 +1,244 @@
+#include "SimCalorimetry/HGCalSimAlgos/interface/HGCalSiNoiseMap.h"
+
+//
```

Comment:
please remove this file "#HGCalSiNoiseMap.cc#" from the PR entirely (including from the history)

Reply:
Hello @kpedro88 , I seem not to be able to remove this file from the history.

I've tried this approach https://blog.liplex.de/remove-file-from-git-history/ 
(in a few variants; the process would take a few days to complete) as well as 
manually  rolling back the history+redoing the commit which accidentally added the file + cherry-pick all following commits 
(leading to "Updates were rejected because the tip of your current branch is behind" when pushing to github)

I am out of ideas. Do you have tips ?
Simply removing the file it's trivial and can be done right away
Thanks!